requirements_dev.in

```diff
@@ -0,0 +1,10 @@
+# Shared/Runtime dependencies
+
+-r requirements.in
+
+# Development-only dependencies
+
+
+# Lint-only dependencies
+
+black==20.8b1
```

Comment:
Is this the standard `requirements_dev.in` template?

Reply:
Not quiet;

```
pip-compile --output-file=requirements-dev.txt requirements-dev.in
```

would produce 

```
#
# This file is autogenerated by pip-compile
# To update, run:
#
#    pip-compile --output-file=requirements-dev.txt requirements-dev.in
#
appdirs==1.4.4            # via black
black==20.8b1             # via -r requirements-dev.in
certifi==2020.6.20        # via requests
chardet==3.0.4            # via requests
click==7.1.2              # via black
idna==2.10                # via requests
mypy-extensions==0.4.3    # via black
pathspec==0.8.0           # via black
regex==2020.9.27          # via black
requests==2.24.0          # via -r requirements.in
toml==0.10.1              # via black
typed-ast==1.4.1          # via black
typing-extensions==3.7.4.3  # via black
urllib3==1.25.10          # via requests
``

However, adding `-r requirements.in` to the start of the dev dependencies avoids listing the entire contents of `requirements.txt`. 

The comments that appear after this line are not necessary (personal preference) but it helps provide a logical grouping to identify what dependencies belong too. Useful if/when more are added.

```
# Development-only dependencies


# Lint-only dependencies

black==20.8b1
```

I'm happy to make changes around any of this if it feels inappropriate for the project.